lib/cms-providers/prismic/custom_types/stage.json

```diff
@@ -1,46 +1,46 @@
 {
-  "Main": {
-    "name": {
-      "type": "StructuredText",
-      "config": {
-        "single": "heading1",
-        "label": "Name"
+  "Main" : {
+    "name" : {
+      "type" : "StructuredText",
+      "config" : {
+        "single" : "heading1",
+        "label" : "Name"
       }
     },
-    "uid": {
-      "type": "UID",
-      "config": {
-        "label": "Slug"
+    "uid" : {
+      "type" : "UID",
+      "config" : {
+        "label" : "Slug"
       }
     },
-    "stream": {
-      "type": "Link",
-      "config": {
-        "label": "Stream",
-        "select": "web"
+    "stream" : {
+      "type" : "Link",
+      "config" : {
+        "label" : "Stream",
+        "select" : "web"
       }
     },
-    "discord": {
-      "type": "Link",
-      "config": {
-        "label": "Discord",
-        "select": "web"
+    "discord" : {
+      "type" : "Link",
+      "config" : {
+        "label" : "Discord",
+        "select" : "web"
       }
     },
-    "schedule": {
-      "type": "Group",
-      "config": {
-        "fields": {
-          "talk": {
-            "type": "Link",
-            "config": {
-              "select": "document",
-              "customtypes": ["talk"],
-              "label": "Talk"
+    "schedule" : {
```

Comment:
Where these spaces intentionally added?

Reply:
The spaces were not an intentional add, it was generated that way when exported from Prismic.